__tests__/lib/index.test.js

```diff
@@ -37,18 +58,14 @@ describe( 'go-search-replace', () => {
 	} );
 	describe( 'replace()', () => {
 		it( 'returns an instance of the stdout object', async () => {
-			const binary = process.env.CI === 'true' ? './bin/go-search-replace-test' : null;
-			const result = await replace( readableStream, [ 'thisdomain.com', 'thatdomain.com' ], binary );
-			await new Promise( resolve => {
-				writeableStream.on( 'finish', () => {
-					resolve();
-				} );
-
-				result.pipe( writeableStream );
-			} );
-
-			const outFile = fs.readFileSync( writeFilePath ).toString();
+			const { outFile } = await testHarness( [ 'thisdomain.com', 'thatdomain.com' ] );
 			expect( outFile ).toContain( 'thatdomain.com' );
```

Comment:
For completeness, this test and the next one should verify that the opposite domain is not included in outFIle either.

Reply:
Makes sense, I can add that.  Thanks Mo 👍